### PR TITLE
Change package manager yarn to npm

### PR DESCRIPTION
**Pull request by uidoyen**

### CODE REVIEW

**Review by indraraj — APPROVED** [review]
I am getting errors in resolving peer dependency on my local. But looks like it can be my local issue since PR check is passing it.

File: ui/README.md [hunk 1]
```diff
@@ -2,44 +2,36 @@
...
 
 ```sh
-yarn install
+npm install
 ```
 
 Run a full build
 ```sh
-yarn build
+npm build
```
Comment (rk3rn3r) [inline ui/README.md:21]: npm run build?

File: ui/packages/ui/package.json [hunk 2]
```diff
@@ -13,15 +13,15 @@
     "start": "webpack serve --color --progress --config webpack.dev.js",
     "start:prod": "node scripts/serve",
     "lint": "tslint -c ./tslint.json --project .",
-    "lint:fix": "yarn run lint --fix",
+    "lint:fix": "npm run lint --fix",
     "format": "prettier --check --write ./src/**/*.{tsx,ts}",
     "format:fix": "prettier --write --ignore-unknown 'src'",
     "build:bundle-profile": "webpack --profile --json > stats.json",
-    "bundle-profile:analyze": "yarn build:bundle-profile && webpack-bundle-analyzer ./stats.json",
+    "bundle-profile:analyze": "npm build:bundle-profile && webpack-bundle-analyzer ./stats.json",
```
Comment (rk3rn3r) [inline ui/packages/ui/package.json:20]: `npm run` here too?

File: .github/workflows/backend-testing.yml [hunk 3]
```diff
@@ -43,16 +43,7 @@ jobs:
...
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-      - name: Cache local Yarn repository
-        uses: actions/cache@v2
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: dbzui-${{ runner.os }}-yarn-${{ hashFiles('dbzui/ui/yarn.lock') }}
-          restore-keys: |
-            dbzui-${{ runner.os }}-yarn-
+
       - name: Build debezium-checkstyle
```
Comment (rk3rn3r) [inline .github/workflows/backend-testing.yml:47]: I think we need to bring back caching? Don't we?

**Comment by rk3rn3r** [comment]
LGTM @uidoyen! Thx a lot for making this possible so quickly!
@indraraj maybe you can try to remove all local `node_module` dirs and try re-run? Please double-check that everything is working and then please merge the PR!

**Comment by indraraj** [comment]
> @indraraj maybe you can try to remove all local `node_module` dirs and try re-run? 

I had to update the node version to 16

**Pull request merged**